core/main/src/Options/Classes/Options.ts

```diff
@@ -102,12 +105,16 @@ export class Options implements IOptions, IOptionLoader<IOptions> {
         if (data.pauseOnBlur !== undefined) {
             this.pauseOnBlur = data.pauseOnBlur;
         }
-
         this.background.load(data.background);
         this.particles.load(data.particles);
         this.infection.load(data.infection);
         this.interactivity.load(data.interactivity);
         this.backgroundMask.load(data.backgroundMask);
+        if(data.themes !== undefined){
+            for(const theme in data.themes){
```

Comment:
this should be `for (const theme of data.themes) {`

`in` enumerates property, `of` enumerates the array